### PR TITLE
Database improvements

### PR DESCRIPTION
**Pull request by aNullValue**
- Adds a lot of "is_validated" columns, whose value will indicate whether or not their corresponding data columns have been validated by a human.
- Adds functionality to ensure that jurisdiction timezone_default always matches its state timezone_default when state.timezone_enforced is true (ensures that single-timezone states remain consistent).
- Adds functionality to ensure that location timezone always matches its jurisdiction timezone_default when jurisdiction.timezone_enforced is true (ensures that single-timezone EAJs remain consistent).
- Adds example PostGIS enablement script for convenience of anyone who might be running migrations manually on a fresh database.
- Amends example data to provide slightly better instruction in README, and to conform to revised schema.

### CODE REVIEW

File: backend/migrations/sql/20200925163259-postgis.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-- this file provided for convenience only, and would need to be run in a pg instance that is not prebuilt w/ postgis extensions
```
Comment (jafow) [inline backend/migrations/sql/20200925163259-postgis.sql:1]: I don't think we should run with the migrations. These are setup in the cloud instance on creation and the docker pg container is pre-built too
https://github.com/hackforla/ballotnav/blob/master/backend/db/create-postgis-extension.sql

I'd recommend adding a drop before if we keep in migration

**Review by jafow — APPROVED** [review]
👍

File: backend/migrations/sql/20200925163259-postgis.sql (hunk 1 above)
Comment (aNullValue) [reply backend/migrations/sql/20200925163259-postgis.sql:1]: The sequelize migrations (intentionally) never call this code, for exactly that purpose.

Comment (jafow) [reply backend/migrations/sql/20200925163259-postgis.sql:1]: Cool that's great (but why's it in the migrations folder?) 😉

**Pull request merged**